packages/outputarea/src/widget.ts

```diff
@@ -908,14 +908,22 @@ export interface IStdin extends Widget {
  */
 export class Stdin extends Widget implements IStdin {
   private static _history: string[] = [];
+  private static _placeholder = '↑↓ for history. Search history with c-↑/c-↓';
```

Comment:
See below, we need this in the place where it is used.

Reply:
done